### PR TITLE
Fix broken Githun action ID

### PR DESCRIPTION
**Pull request by d4hines**
## Problem

Observe that our Docker CI is failing because we have multiple builds with id `docker_build`: https://github.com/marigold-dev/deku/actions/runs/2059345040

## Solution

- change the id's to be unique
- ~disable the docker static build, since it's broken anyway right now.~

### CODE REVIEW

**Comment by EduardoRFS** [comment]
Please, just do the proper thing. https://github.com/marigold-dev/deku/pull/522

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -23,24 +23,25 @@ jobs:
...
+      #   with:
+      #     file: ./docker/static.Dockerfile
+      #     context: .
+      #     platforms: linux/amd64
+      #     cache-from: type=gha
+      #     cache-to: type=gha,mode=max
+      #     push: ${{ github.ref == 'refs/heads/main' && github.event_name == 'push' }}
+      #     tags: |
+      #       ghcr.io/marigold-dev/deku:latest
+      #       ghcr.io/marigold-dev/deku:${{ github.sha }}
```
Comment (ulrikstrid) [inline .github/workflows/docker.yml:40]: If we uncomment this we can merge it

**Review by ulrikstrid — CHANGES_REQUESTED** [review]
Since #522 fixes the static build we can un-comment but keep the changed id

File: .github/workflows/docker.yml (hunk 1 above)
Comment (ulrikstrid) [reply .github/workflows/docker.yml]: I took the liberty to do this

**Review by ulrikstrid — APPROVED** [review]

**Comment by ulrikstrid** [comment]
Since this PR is strictly better than what we have in `main` I think we should merge it so that we can keep moving on the testnet before merging the new static build

**Comment by d4hines** [comment]
superseded by #522

**Pull request closed without merging**